pkg/compare/testdata/ReferenceV2InlineRegex/localwithDiffout.golden

```diff
@@ -0,0 +1,22 @@
+**********************************
+
+Cluster CR: v1_ConfigMap_kubernetes-dashboard_kubernetes-dashboard-settings
+Reference File: cm-regex-with-diff.yaml
+Diff Output: diff -u -N TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings
+--- TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings	DATE
++++ TEMP/v1_configmap_kubernetes-dashboard_kubernetes-dashboard-settings	DATE
+@@ -8,4 +8,4 @@
+ spec:
+   bigTextBlock: |-
+     This is a big text block with some static content, like this line.
+-    It also has a place where (?<username>[0-9]+) would put in their own name. (?<username>[0-9]+) would put in their own name.
++    It also has a place where exampleuser would put in their own name. exampleuser would put in their own name.
```

Comment:
Is there a way we could make it more clear that the diff is because of the regex mismatch and not some other difference?  For example, could we expand the template side to have something like `<<exampleuser does not match (?<username>[0-9+)>>`?

Reply:
I think that should be in a future story.

Reply:
regex match won't tell you where it failed so you would probably have to enumerate the sub expressions until something failed.

Reply:
Agreed.  Future PR.